### PR TITLE
Fixed compilation error in tests.cpp when not building with TESTS=1

### PR DESCRIPTION
**Pull request by Narasimha1997**
Compilation fails when building without `TESTS` enabled, because we are not using `std` namespace. (You can have a look at `#else` part of `src/tests.cpp`. This can be fixed by adding

```
using namespace std;
```
So `cout` and `endl` symbols are imported properly during compilation.

This patch fixes the above error.

### CODE REVIEW

**Comment by vvaltchev** [comment]
Hi Narasimha,
Thanks for reporting this problem! I'm happy that you're interested in this project as well :-)

Getting to the point: `using namespace std` is generally considered a "bad practice"
in C++. See: https://stackoverflow.com/questions/1452721/

While it's OK for very small projects (I used it in `mylang` until recently), the
project now seems too big for that and I spent some effort removing it from everywhere.

While doing that, I introduced the file `defs.h` with some `using` statements like:

```C++
using std::cin;
using std::cout;
using std::cerr;
using std::ostream;
using std::endl;
```
And, in theory, I included the file "everywhere". The idea is that only a very limited set of types and functions
in `std` are freely usable without the namespace prefix. The rest, require the `std::` prefix especially in header
files (.h). In source files instead (.cpp, .cpp.h), it's possible to add additional `using std::<something>` statements
at the beginning of the file, but that's OK only if there *many* instances of a given type.

In conclusion, the right fix here is to just add `#include "defs.h"` before the first `#include <iostream>` in tests.cpp.
Would you like to update your pull request this way?

Thanks,
Vlad

**Comment by Narasimha1997** [comment]
@vvaltchev , Hi, Thanks,  I have made the necessary changes in the latest commit.

Also, another issue, the interpreter throws syntax error for `samples/fib`, it cannot handle multiple variable declarations in the same line. To be more specific, the syntax error is in line 5 of `samples/fib`

**Comment by vvaltchev** [comment]
@Narasimha1997 thanks for reporting this as well. I just fixed it.

What happened: initially, `mylang` supported that classic syntax:

```
var a=1,b=2,c=3;
```

But later I decided that's incompatible with a new nicer feature called "array expansion",
so I dropped it and I forgot to fix the sample. The array expansion feature allows things like:

```
var a,b = [3, 10];
a, b += [1, 2]; # increase a by 1 and b by 2
a, b += 10; # increase both a and b by 10
```

Which is super-cool to me, especially in `foreach` loops. If you're curious, check the `readme`
branch, where I'm writing some documentation in the README.md file.
